### DISCUSSION TITLE
Is JRuby-9.2.16.0 compatible with Ruby-2.5.8?

### DISCUSSION THREAD

**Discussion by tedt10e** [jruby/jruby#6611] [Q&A] [answered]
Hello JRuby Friends,

Since this is not an issue, I believed using Discussions is more appropriate. 😁

Is JRuby-9.2.16.0 compatible with Ruby-2.5.8 or only compatible with 2.5.7 as printed from ```ruby -v```?

```jruby 9.2.16.0 (2.5.7) 2021-03-03 f82228dc32 OpenJDK 64-Bit Server VM 15.0.2+7-27 on 15.0.2+7-27 +jit [darwin-x86_64]```

Many thanks.

**Comment by tedt10e** [comment]
Hi @headius 

Can you please advise?

Many Thanks. 🙏

**Comment by headius** [comment]
Sorry I thought this was an issue and then could not find it!

JRuby's reported Ruby compatibility level is a little "soft" in that it mostly represents the stdlib files plus some measure of feature compatiblity. We have not updated stdlib to include 2.5.8 changes, and that probably should happen. I will investigate.

**Comment by headius** [comment, marked as answer]
This will be addressed in JRuby 9.2.17.0 by #6624.

**Reply by tedt10e** [reply]
Thank you so much 🙏 @headius
